solidity/test/bridge/Bridge.test.ts

```diff
@@ -188,6 +188,87 @@ describe("Bridge", () => {
     redemptionTimeout = (await bridge.redemptionParameters()).redemptionTimeout
   })
 
+  describe("updateDepositParameters", () => {
```

Comment:
`Bridge.Parameters.test.ts` please.

Reply:
Done in 49f3a5d7a6cabc18a383e4b72c7f4077cdf79499.